### DISCUSSION TITLE
Image Format ImageProV3

### DISCUSSION THREAD

**Discussion by flyman02** [keirf/greaseweazle#404] [Ideas]
Hi, I found a series of floppy images generated by an old DOS ImageProvro3 program.
I wanted to recreate the floppy disks, but with the dos emulated environment it doesn't work. I thought I would do it with GW.EXE but I don't know if this hip format is supported. To create the image it also works from emulated dos.
Attached the prog dos (the zip extension must be renamed)

[ImageProV3.ODT](https://github.com/keirf/greaseweazle/files/14113122/ImageProV3.ODT)

**Comment by keirf** [comment]
An example image would be more useful. I have no way to usefully run the ImagePro software.

**Reply by flyman02** [reply]
Sorry, I thought it was more useful in software. Here is an image to rename in IMG:
[IMAGE720.ODT](https://github.com/keirf/greaseweazle/files/14128255/IMAGE720.ODT)

**Reply by flyman02** [reply]
Another example:
[IMGPRO_1.ODT](https://github.com/keirf/greaseweazle/files/14128333/IMGPRO_1.ODT)

**Reply by flyman02** [reply]
I created an image from the Imagepro program and I also attach the standard image of the original disk (this time it should be renamed in zip)
[Image Test.ODT](https://github.com/keirf/greaseweazle/files/14128436/Image.Test.ODT)
